superset-frontend/src/components/Chart/Chart.tsx

```diff
@@ -150,9 +153,16 @@ const MonospaceDiv = styled.div`
   overflow-x: auto;
   white-space: pre-wrap;
 `;
+class Chart extends React.PureComponent<ChartProps, {}> {
+  static defaultProps = defaultProps;
+
+  renderStartTime: any;
 
-class Chart extends React.PureComponent {
-  constructor(props) {
+  renderContainerStartTime: number;
```

Comment:
Is `renderContainerStartTime` actually used anywhere?

Reply:
I removed it, thank you